ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlannerViewModel.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using CommunityToolkit.Mvvm.Input;
+using ElectronicObserver.Common;
+using ElectronicObserver.Data;
+using ElectronicObserver.Services;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Mocks;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+
+public partial class EquipmentUpgradePlannerViewModel : WindowViewModelBase
+{
+	private ObservableCollection<EquipmentUpgradePlanItemViewModel> PlannedUpgrades { get; set; } = new();
+	public ObservableCollection<EquipmentUpgradePlanItemViewModel> PlannedUpgradesFilteredAndSorted { get; set; } = new();
+
+	public EquipmentUpgradePlannerTranslationViewModel EquipmentUpgradePlanner { get; set; } = new();
+
+	private readonly EquipmentPickerService EquipmentPicker;
```

Comment:
Use get only properties instead of readonly fields. You can see references to properties easier, and they can be debugged.